build.gradle

```diff
@@ -77,6 +77,9 @@ dependencies {
     compile "net.minidev:json-smart:2.4.6" // Pinning this above the transitive version from json-path to get CVE fix
     compile "org.ow2.asm:asm:9.1"
     compile "org.slf4j:slf4j-api:1.7.30"
+    compile "org.apache.logging.log4j:log4j-slf4j-impl:2.13.2"
```

Comment:
log4j shouldn't be made a production dependency. At present it's only used for testing. Is there a reason you're adding it to the compile scope in addition to test?